danndav/bhankas
# #165481026: setting up databaseserver

What does this PR do?
This PR shows setup of database

Description of Task to be completed?
set up database using postgresql
use a pool connection for the database

How should this be manually tested?
git fetch and checkout toch-165481026-setting-up-databaseserver
checkout to the branch
run the application
navigate to /task/database connection file
click on it  to check changes

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
#165481026

screenshots (if appropriate)
N/A

Questions:
N/A\
